modules/identity-provider-parameters.adoc

```diff
@@ -5,7 +5,7 @@
 [id="identity-provider-parameters_{context}"]
 = Identity provider parameters
 
-Four parameters are common to all identity providers:
+Several parameters are common to all identity providers:
```

Comment:
How about _The following paramters_? 
Wonder what happened to the other two?

Reply:
I like `The following parameters`.

I believe that `claim` and `lookup` were removed, as these could be deduced by the engineering team depending on the identity provider; so these exist, but are not configured by the customer and are instead inferred.